### PR TITLE
Updated all projects for Xcode 9.3/Swift 4.1.

### PR DESCRIPTION
**Pull request by E-B-Smith**

### CODE REVIEW

**Comment by BranchMacMini** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/qabot.stage.branch.io/352212df632dd908750f5a80b2220c0b/352212df632dd908750f5a80b2221a64)
---
*Duration*: 18 minutes and 32 seconds
*Result*: All 227 tests passed, but please **fix 2 warnings**.
*Test Coverage*: 62%

**Comment by E-B-Smith** [comment]
@sojanpr Can you thumb this?

File: Branch-TestBed-Swift/TestBed-Swift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (Sarkar) [inline Branch-TestBed-Swift/TestBed-Swift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Do we need this file? I'm not sure if we do

File: Branch-TestBed-iMessage/Branch-TestBed-iMessage.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```
Comment (Sarkar) [inline Branch-TestBed-iMessage/Branch-TestBed-iMessage.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Same with this one

File: Branch-TestBed/Branch-TestBed-UITests/UITestBed.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (Sarkar) [inline Branch-TestBed/Branch-TestBed-UITests/UITestBed.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Same as above

File: Branch-TestBed/Branch-TestBed.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```
Comment (Sarkar) [inline Branch-TestBed/Branch-TestBed.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Same

**Comment by Sarkar** [comment]
tested on my local, looked good. no warnings as described

File: Branch-TestBed-Swift/TestBed-Swift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (E-B-Smith) [reply Branch-TestBed-Swift/TestBed-Swift.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Yes, we want it.  Per Apple:

Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace. (37293167)

**Pull request merged**